kickstarter/android-oss
# MBL-667: new event add_payment_method

# 📲 What

New third party event sent


# 🛠 How
- This specific event, is triggered when a new payment method is added on the pledge screen, or when the selected payment method changes.


# 👀 See
Network request param
![success](https://github.com/kickstarter/android-oss/assets/4083656/b4b4fd81-eb90-47ae-a6cc-e2a481d96d5b)
eters sent 



Response from the mutation
<img width="678" alt="success2" src="https://github.com/kickstarter/android-oss/assets/4083656/78e9be4a-871c-4801-accf-772efe5c9355">



|  |  |

# 📋 QA
- Search for the project "be a girl" on staging, using the Android Studio network profiler you can see this event triggered when a payment method is added or the selection changes.
- Make sure you pledge to the basic "reward no reward", the reward shipping addons, and the reward with addOns, the `pledgeAmount` and `shipping` properties will change accordingly.

# Story 📖

[MBL-667](https://kickstarter.atlassian.net/browse/MBL-667)


[MBL-667]: https://kickstarter.atlassian.net/browse/MBL-667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
